scripts/update.sh

```diff
@@ -23,6 +23,9 @@ log_warn "\nBacking up deployment directory in ${backup_dir}"
 cp -r "${DEPLOYMENT_DIR}" "${backup_dir}" || fn_die "\nError: could not backup deployment directory. Fix it before proceeding any further. Exiting...\n"
 cp "${ROOT_DIR}/compose_files/docker-compose-${NODE_TYPE}.yml" "${DEPLOYMENT_DIR}/docker-compose.yml"
 
+# Migration "NH_" -> "ZKV_" (TODO: remove in next release)
```

Comment:
Do you still need this comment?

Reply:
The `TODO` is for the next release, since no other renaming (and consequent migrations) is expected.